web-admin/src/features/billing/banner/BillingBannerManager.svelte

```diff
@@ -12,7 +16,11 @@
     const changedOrganization =
       !from || !to || from.params.organization !== to.params.organization;
     if (changedOrganization) {
-      eventBus.emit("banner", null);
+      eventBus.emit("banner", {
+        id: BillingBannerID,
+        priority: BillingBannerPriority,
```

Comment:
We shouldn't need to specify the priority when clearing the banner. So maybe this merits a new event `remove-banner`.